docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+
+Edit the workflow configuration file
+------------------------------------
+
+This time, you will make some modifications to the pre-written configuration file ``genomic-surveillance.yaml`` before running the workflow. The contents are copied below (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/reference/aligned.fasta.xz
+     - name: custom_data
+       metadata: data/idaho.metadata.tsv
+       sequences: data/idaho.sequences.fasta
+     - name: background_data
+       metadata: https://data.nextstrain.org/files/ncov/open/north-america/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/north-america/aligned.fasta.xz
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     idaho:
+       title: "Idaho-specific genomic surveillance build"
+       subsampling_scheme: idaho_scheme
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+   subsampling:
+     idaho_scheme:
+       custom_sample:
+         query: --query "(custom_data == 'yes')"
+         max_sequences: 5000
+       usa_context:
+         query: --query "(background_data != 'yes') & (country == 'USA')"
+         max_sequences: 1000
+         group_by: division year month
+         priorities:
+           type: proximity
+           focus: custom_sample
+       global_context:
+         query: --query "(reference_data == 'yes')"
+         max_sequences: 1000
+         priorities:
+           type: proximity
+           focus: custom_sample
+
+This is similar to the previous file, with some differences:
+
+1. The file paths in the second input are changed to ``idaho.metadata.tsv`` and ``idaho.sequences.fasta``.
+2. There is an additional input ``background_data`` for a regional North America dataset built by the Nextstrain team, for additional context.
+3. The output dataset is renamed ``idaho``, representative of the new custom data in the second input.
+4. There is a new ``subsampling`` section that provides a subsampling scheme ``idaho_scheme`` consisting of three subsamples:
+
+   1. ``custom_sample``
+
+      - This selects sequences from the ``custom_data`` input, up to a maximum of 5000 sequences.
+
+   2. ``usa_context``
+
+      - This selects sequences from the ``background_data`` input, up to a maximum of 1000 sequences.
+      - Sequences are subsampled evenly across all combinations of ``division``, ``year``, ``month``, with sequences genetically similar to ``custom_sample`` prioritized over other sequences.
+
+   3. ``global_context``
+
+      - This selects sequences from the ``reference_data`` input.
+
+This configuration file works without modifications, but the following steps guide you to customize names and titles, if appropriate:
```

Comment:
What if we moved this line right after the section header above (`Edit the workflow configuration file`), so users know early on that they can run these commands without modifying a YAML if they want.

Reply:
No longer necessary with a709eccd, based on https://github.com/nextstrain/ncov/pull/894#discussion_r832713912